library-no-op/build.gradle

```diff
@@ -15,6 +15,7 @@ android {
     lintOptions {
         warningsAsErrors true
         abortOnError true
+        disable 'GradleDependency'
```

Comment:
Why do we need this? To not show warnings about future Gradle versions, like we have now?

Reply:
> Why do we need this?

We need this as currently lint is failing on master (as we don't run it on CI) because of this.
In my change I'm suggesting to run `./gradlew build` in the readme, but that command will actually fail as of now.

Either we suppress `GradleDependency` globally or we have to comment every dependency in the .gradle file with `//noinspect GradleDependency`. I generally disable this Lint check and let Gradle (or the other tools like Dependabot) take care of the dependencies.

Anyway I'm amending this change as I'll take care of the lint failures in another PR